Formula/hyper.rb

```diff
@@ -0,0 +1,23 @@
+class Hyper < Formula
+  desc "Hyper_(make VM run as container) client for https://www."
```

Comment:
[Hyper_](https://hyper.sh) is a container-native cloud, where secure containers replace VMs as the infrastructure building block. This package is the CLI's Mac build.  The length of this description was limited by 80 characters, so I added more introduction in this.